### ISSUE TITLE
Auth0-PHP@5.3.1 breaks Laravel-Auth0

### ISSUE THREAD

**Issue by SamuelWillis** [auth0/laravel-auth0#108]
Installing this package installs auth0@5.3.1.

This is causing an exception to be thrown that stems from the `LoginServiceProvider`:

```
[2018-10-31 22:21:05] local.ERROR: Undefined index: environment {"exception":"[object] (ErrorException(code: 0): Undefined index: environment at /var/www/vendor/auth0/auth0-php/src/API/Helpers/InformationHeaders.php:128)"} []
[2018-10-31 22:21:05] local.ERROR: Undefined index: environment {"exception":"[object] (ErrorException(code: 0): Undefined index: environment at /var/www/vendor/auth0/auth0-php/src/API/Helpers/InformationHeaders.php:128)"} []

In InformationHeaders.php line 128:

  Undefined index: environment
```

I've traced it down to the `InformationHeaders::Extend` function that looks to fill the headers in with the old headers `environment` field.

It gets the old headers from the auth0 ApiClient `getInfoHeadersData`. 
[This change to ApiClient](https://github.com/auth0/auth0-PHP/compare/5.3.0...5.3.1#diff-2a2f8d79914bf4c1b66dbfcb46bdb788L42) no longer sets an `environment` field, however.

One suggestion would be to change the `composer.json` to:

```
   "require": {
        "auth0/auth0-php": "=>5.1 <5.3.1",
},
```

I can also raise this issue on the `auth0-php` package as the update to the `ApiClient` has broken the `InformationHeaders@Extend` function (which is slated for depracation).

**Comment by joshcanhelp** [comment]
@SamuelWillis - Thank you for this report! I have an [issue open in the PHP SDK](https://github.com/auth0/auth0-PHP/issues/303) and was confused why `Extend` was being called anywhere. Sounds like that issue is probably on Laravel as well. 

Apologies for the trouble here. I'll release a fix for this today!

**Comment by SamuelWillis** [comment]
@joshcanhelp Thank you so much! Let me know if you need anything more from me!

**Comment by joshcanhelp** [comment]
Fixed and released in [5.3.2](https://github.com/auth0/auth0-PHP/releases/tag/5.3.2)
